packages/cactus-cmd-api-server/src/main/typescript/api-server.ts

```diff
@@ -326,6 +326,11 @@ export class ApiServer {
 
     const instanceId = pluginImport.options.instanceId;
     const pluginPackageDir = path.join(this.pluginsPath, instanceId);
+    const pluginVersion = // version of the npm package
```

Comment:
@elenaizaguirre 

Would this work here?

```suggestion
const pluginVersion = pluginImport.options.version ? "@".concat(pluginImport.options.version) : "";
```

Reply:
@petermetz You're right, I thought that the condition would return true if the version was initialized even if it was an empty string

Reply:
@elenaizaguirre I agree with your initial intuition :-) Javascript treats empty strings also as falsy values for some reason which we can exploit here to our advantage.